src/dns/named.rb

```diff
@@ -96,18 +97,22 @@ def render_repo_named_conf
 
       # TODO: These commands will break hosts DNS. Might be a good idea to run
       # similar commands for hosts
-      RESTART_NAMED_CMDS = [
+      START_NAMED_CMDS = [
         'systemctl disable dnsmasq',
         'systemctl stop dnsmasq',
         'systemctl enable named',
-        'systemctl restart named',
       ].freeze
 
-      def restart_named
+      def start_named
         MetalLog.info 'Restarting named'
-        RESTART_NAMED_CMDS.each do |cmd|
+        START_NAMED_CMDS.each do |cmd|
           SystemCommand.run(cmd)
         end
+        restart_named
+      end
+
+      def restart_named
+        SystemCommand.run('systemctl restart named')
```

Comment:
Not a big deal, but you could factor out this string from here and in `START_NAMED_CMDS` to a new constant, to indicate the same command is run in both cases.